docs/objects/Object.htm

```diff
@@ -235,7 +235,7 @@ <h2>Properties</h2>
 <h2>Functions</h2>
 
 <div class="methodShort" id="Object"><h2>Object</h2>
-<p>Creates an Object from a list of property name, value pairs.</p>
+<p>Creates an Object from a list of property name-value pairs.</p>
```

Comment:
The comma was intended to imply that each pair is two parameters, not one (such as `[name, value]`), although maybe not the best idea. (I had been looking at a JavaScript method that takes an *enumerable*/array of "pairs", where each pair is an array.)

Reply:
Well, actually it is clarified directly in the syntax box below how exactly the pairs must be specified. I also noticed that there is quite an inconsistency regarding this writing style throughout the docs:

docs\commands\index.htm:
27: list of property name and value pairs
729: list of key-value pairs

docs\commands\NumPut.htm:
53: <em>Type</em>, <em>Number</em> pairs

docs\commands\WinSetRegion.htm:
32: pairs of <strong>X-Y</strong>

docs\objects\Map.htm:
16, 53, 60, 91, 110, 122: key-value pairs

Actually "name, value" should only be marked somehow that name and value belong together. In this case I would prefer to use quotes rather than format tags like code, em, strong etc., because I find them a bit exaggerated. What do you think about the following:

Creates an Object from a list of property "name, value" pairs.

Reply:
I thought about it while writing it, but honestly couldn't care less. I have no objection to your suggestion.

However, I think the difference between WinSetRegion and the others is mostly due to the actual syntax of the function, not merely a style inconsistency.

Reply:
> I thought about it while writing it, but honestly couldn't care less. I have no objection to your suggestion.

I've changed it.

> However, I think the difference between WinSetRegion and the others is mostly due to the actual syntax of the function, not merely a style inconsistency.

Don't worry. I leave the "inconsistencies" untouched.